orangecontrib/network/widgets/OWNxAnalysis.py

```diff
@@ -330,7 +330,7 @@ def cancel_job(self, name=None):
         # This does not really work because functions called in those
         # threads do not observe the "is_terminated" flag and won't quit
         if name is None:
-            to_stop = [job.method.name for job in self.running_jobs]
+            to_stop = list(self.running_jobs.keys())
```

Comment:
Just a note: `.keys()` is unnecessary, `list(self.running_jobs)` works the same.